recipes/hcrystalball/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hcrystalball" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d46d60b158af1a4f820444bde8028f872f69494a70af95e25fc6fc439ec9b937
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pyscaffold >=3.1a0,<3.2a0
```

Comment:
@chrisburr should we put pyscaffold under host or builde requirements?

Reply:
I'm fairly sure it will be needed in `host`

Reply:
Does the version constraint need to be this strict?

Reply:
@MichalChromcak Just this question ⬆️

Reply:
It's taken from the setup.cfg generated by pyscaffold. Without any, it kept failing, not sure about the wider range availability. Does that restrict the user side anyhow?

Reply:
@chrisburr If we may leave it as is, it would be currently the preffered (easy) way, so the version matches with pyscaffold generated one in the setup.cfg

Reply:
@ocefpaf or @maresb as https://github.com/conda-forge/pyscaffold-feedstock maintainers - what do you suggest concerning version constraints for pyscaffold in this conda-forge package?

Looking at some existing cases like [here](https://github.com/conda-forge/pygeobase-feedstock/blob/734cdedfac41109e8e23cd5642bed2e73bcf798c/recipe/meta.yaml#L20) or [here](https://github.com/conda-forge/smif-feedstock/blob/a3df80b09d1b1c38b7921d7216b15bc9d3dfb575/recipe/meta.yaml#L27) it seems that the common way to constrain the pyscaffold version is like how we also do it here.

Would be nice if this can go in soon.

Reply:
Indeed it needs ``pyscaffold >=3.1a0,<3.2a0`` in host, because it is a constrain in their ``setup.cfg``

Reply:
For this recipe would be nice to add ``pip check``